flexinput/src/main/java/com/lytefast/flexinput/model/Generic.java

```diff
@@ -0,0 +1,21 @@
+package com.lytefast.flexinput.model;
+
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.database.Cursor;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.support.annotation.Nullable;
+
+
+/**
+ * @author Sam Shih
+ */
+public class Generic<T> extends Attachment {
```

Comment:
It can, but since Some attachments like Photo don't actually have a rawData beside the Uri I don't think we should add it.